students/lauraannf/lessons/lesson06/assignment/write_csv.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  2 16:37:33 2019
+
+@author: Laura.Fiorentino
+"""
+
+import csv
+import uuid
+from datetime import date, timedelta
+import random
+
+CSV_DICT = {}
+with open('data/exercise.csv', 'r', newline='') as csvfile:
+    EX_FILE = csv.reader(csvfile)
+    N = 1
+    for row in EX_FILE:
+        CSV_DICT[N] = {'guid': row[0], 'num1': row[1], 'num2': row[2],
+                       'num3': row[3], 'num4': row[4], 'date': row[5],
+                       'ao': row[6]}
+        N += 1
+
+
+def new_date(rand):
+    """create a new date"""
+    date1 = date(2000, 1, 1)
+    new_date_random = date1 + timedelta(rand)
+    return new_date_random.strftime("%m/%d/%Y")
+
+
+def ao_choice(choice):
+    """ pick ao or no ao"""
+    if choice == 1:
+        return 'ao'
+    else:
```

Comment:
This _else_ is unnecessary, as it is preceded by a _return_ statement, so the only way _return ''_ could happen is if _return 'ao'_ didn't. No need to fix this one, just something to keep in mind when using if statements that contain a return.